os/board/imxrt1050-evk/src/imxrt_boot.c

```diff
@@ -290,14 +293,18 @@ void board_initialize(void)
 
 #ifdef CONFIG_IMXRT_TIMER_INTERFACE
 	{
-		int timer_idx;
+		int timer_idx = 0;
 		char timer_path[CONFIG_PATH_MAX];
 
 #ifdef CONFIG_IMXRT_GPT
 		for (timer_idx = 0; timer_idx < IMXRT_GPT_CH_MAX; timer_idx++) {
 			snprintf(timer_path, sizeof(timer_path), "/dev/timer%d", timer_idx);
 			imxrt_timer_initialize(timer_path, timer_idx);
 		}
+#endif
+#ifdef CONFIG_IMXRT_PIT
+		snprintf(timer_path, sizeof(timer_path), "/dev/timer%d", timer_idx);
```

Comment:
1.  If it has only one pit, we don't need to use this snprintf.
2. As qtmr does, let's use different name.
timer -> pit

Reply:
ok done.